spec/venue_spec.rb

```diff
@@ -44,12 +39,48 @@
 
   describe '#yell_at_patrons' do
     it 'returns a list of uppercased names' do
-      skip
       venue = Venue.new('Bluebird', 4)
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
       expect(venue.yell_at_patrons).to eq ['MIKE', 'MEGAN', 'BOB']
     end
   end
+
+  describe 'iteration 3' do
```

Comment:
Nice job practicing using a `before(:each)`. Your testing looks very clean and readable.